_includes/collapse.html

```diff
@@ -0,0 +1,20 @@
+<!-- Start Collapse.html -->
```

Comment:
Can we use a pure-css based collapsible instead?

https://www.digitalocean.com/community/tutorials/css-collapsible

Reply:
Changed to css-collapsible in https://github.com/subnero1/subnero1.github.io/pull/120/commits/aba146c0e2624dd6af0b635d60c3f3681ffe30bd and  https://github.com/subnero1/subnero1.github.io/pull/120/commits/bab7b4bc084ab549d7cccbfe82cb8f0c43f73534.